src/compiler/lsp/protocol.h

```diff
@@ -0,0 +1,130 @@
+// Copyright (C) 2019 Toitware ApS.
```

Comment:
New.
We introduce an abstraction for talking to the Lsp Server.
Previously, code would just write to stdout.
Now we go through the protocol. Eventually we want to be able to change the protocol in one place.